probe.c

```diff
@@ -21,16 +21,18 @@
 #include "system.h"
 #include "probe.h"
 #include "counters.h"
-
+#include "settings.h"
```

Comment:
What was the reason for adding all these includes if they don't get used in this file?